### PR TITLE
chore: move imports to top of files

### PR DESCRIPTION
**Pull request by liammoat**
## Purpose

* Update a few occurrences where imports were [outside top level](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/import-outside-toplevel.html). 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix (`fix`)
[ ] Feature (`feat`)
[x] Code style update (formatting, local variables) (`style`)
[ ] Refactoring (no functional changes, no api changes) (`refactor`)
[ ] Documentation content changes (`docs`)
[ ] Other... Please describe: (`perf`, `test`, `build`, `ci`, `chore`, `revert`)
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Run the test suite

```
poetry run pytest -m "not azure"
```

## What to Check
Verify that the following are valid
* Tests run successful and import modules as expected

## Other Information
This change also introduced some circular dependencies in the strategy files. Some symbols were moved to their own files to remove this coupling.

### CODE REVIEW

File: code/backend/batch/utilities/orchestrator/OrchestrationStrategy.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class OrchestrationStrategy(Enum):
```
Comment (gaurarpit) [inline code/backend/batch/utilities/orchestrator/OrchestrationStrategy.py:4]: this is nice. :)

File: code/backend/batch/utilities/document_chunking/ChunkingStrategy.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+
+
+class ChunkingStrategy(Enum):
```
Comment (gaurarpit) [inline code/backend/batch/utilities/document_chunking/ChunkingStrategy.py:4]: this is really neat.

File: code/backend/batch/utilities/orchestrator/__init__.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 from typing import List
 import os.path
 import pkgutil
-from .Strategies import OrchestrationStrategy
+from .OrchestrationStrategy import OrchestrationStrategy
 
 
 class OrchestrationSettings:
```
Comment (gaurarpit) [inline code/backend/batch/utilities/orchestrator/__init__.py:8]: minor/low-priority: I would also suggest to move the OrchestrationSettings class into OrchestrationSettings file ( snake_case_named or whatever for now) and out of init.py as it's really hidden inside and impacts readability.

**Review by gaurarpit — APPROVED** [review]
looks much cleaner. LGTM.
one minor suggestion, but not a blocker.

File: code/backend/batch/utilities/document_loading/Strategies.py [hunk 4]
```diff
@@ -10,20 +14,12 @@ class LoadingStrategy(Enum):
 
 def get_document_loader(loader_strategy: str):
```
Comment (ross-p-smith) [inline code/backend/batch/utilities/document_loading/Strategies.py:15]: I feel that this should really be a static `DocumentLoadingFactory.create(loader_strategy: str)` - but feel might be pushing it for this PR - (feel free to resolve!)

**Review by ross-p-smith — APPROVED** [review]
Many thanks!

File: code/backend/batch/utilities/orchestrator/__init__.py (hunk 3 above)
Comment (liammoat) [reply code/backend/batch/utilities/orchestrator/__init__.py:8]: Thanks Arpit. I agree with your comment but, on this occasion, I am going to leave this as-is - in a hope to keep the diff on this PR minimal.

**Pull request merged**